examples/metal-nimf-2-service-profile-connection/main.tf

```diff
@@ -38,3 +38,12 @@ module "metal_2_service_profile" {
   zside_location        = var.zside_location
   zside_fabric_sp_name  = var.zside_fabric_sp_name
 }
+
+resource "time_sleep" "wait_connection" {
+  depends_on = [module.metal_2_service_profile]
+  create_duration = "2m"
+}
+
+data "equinix_metal_connection" "NIMF-test" {
```

Comment:
Add depends_on = [time_sleep.wait_dl_connection] line in Metal Connection data source block